dietpi/dietpi-software

```diff
@@ -6800,28 +6801,40 @@ _EOF_
 			Download_Test_Media
 		fi
 
-		software_id=68 # Remot3.it
+		software_id=68 # Remote.It
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Installing
 
+			local licensekey=
+            G_WHIP_BUTTON_CANCEL_TEXT='Skip'
+			G_WHIP_INPUTBOX 'If you do not already have the Remote.It Desktop application on your laptop, install it.
+https://www.remote.it/download\n
+Please enter your license key obtained from the Remote.It Desktop application.
+[Account] -> [Licensing]\n\n
+NB: This is optional, select "Skip" to leave it empty.
+    If so, register the device using the claim code shown during the installation process.\n
+    See below for instructions on how to register the device using a claim code:
+    https://docs.remote.it/software/device-package/installation#3.-claim-and-register-the-device\n' && licensekey=$G_WHIP_RETURNED_VALUE
+            G_EXEC export R3_REGISTRATION_CODE="$licensekey"
+            G_EXEC export R3_PLATFORM_CODE="1072"
+
 			# ARMv6/7
-			local arch='armhf'
+			local url='https://downloads.remote.it/remoteit/v4.15.2/remoteit-4.15.2.armhf.deb'
```

Comment:
Is there a way to obtain the latest reversion automatically, so what we do not need to review and update the version on every DietPi release + users get possibly important updates as fast as they are available.

Reply:
We will prepare a URL from which you can retrieve the latest version of the package. Please wait a moment.

Reply:
Any news on the download link? 🙂

Reply:
It will be ready shortly.

Reply:
I have modified the download link.